tests/utils.py

```diff
@@ -0,0 +1,13 @@
+from sqlalchemy.orm import Session
```

Comment:
Why have you added an empty `conftest.py` file?

Reply:
this is not my work.

Reply:
I will delete it although.